docs/100-upgrade/migration-guides/3.5-3.6.mdx

```diff
@@ -123,3 +123,23 @@ index 7783303fd..a8f621d20 100644
          },
          onError(error: unknown) {
 ```
+## Payzen Module
+
+In this version, we introduced Payzen compatibility for Magento 2. You now need to pass a parameter to the Payzen extension in your `.front-commerce.config.ts` file:
+
+```ts title=".front-commerce.config.ts"
+// ...
+import payzen from "@front-commerce/payzen";
+
+export default defineConfig({
+  // ...
+  extensions: [
+    // ...
+    // For Magento 1 usage
+    payzen("front-commerce-magento1")
+    // For Magento 2 usage
+    payzen("front-commerce-magento2")
```

Comment:
**nitpick:** projects already using the extension before 3.6 were likely using Magento1

**suggestion:** document what **they** have to do (ideally it should have been a codemod instead)


```suggestion
    payzen("front-commerce-magento1")
```

Reply:
I don't understand? previously there was no notion of flavor: https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/payzen/index.ts?ref_type=heads#L7 so it would have been `payzen()`

Reply:
Ah never mind, I understand now.